src/SpeciesElements.f90

```diff
@@ -10,45 +10,34 @@
 !> @brief Utility module for coverting different forms of elements and isotopes
 !>
 !> This package provides an interface to convert a chemical formula into a
-!> (0:118) size array.
+!> (1:119) size array.
 !>
 !++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++!
 MODULE SpeciesElementsModule
   USE IntrType
   USE Strings
   USE ElementsIsotopes
+  USE IO_Strings
   IMPLICIT NONE
   PRIVATE
 
-  ! Member
-  PUBLIC :: SpeciesElementsType
-
-  !> Type that converts a given species chemical formula and molar amount
-  !! to an array of molar amounts ordered by atomic number 
-  TYPE :: SpeciesElementsType 
-
-  !List of type bound procedures
-  CONTAINS
-    PROCEDURE,PASS :: getElementArray
-    PROCEDURE,PASS,PRIVATE :: isChar
-    PROCEDURE,PASS,PRIVATE :: isCharCap
-    PROCEDURE,PASS,PRIVATE :: isCharLow
-  ENDTYPE SpeciesElementsType
+  !List of Public Members
+  PUBLIC :: getElementArray
```

Comment:
I should have specified this before, but this should have some reference to the module it lives in.  Let's rename to `SpeciesElements_getElementArray`

Reply:
Seems long but i changed it